apps/mobile/src/features/balances/bitcoin/bitcoin-balance.tsx

```diff
@@ -31,13 +32,21 @@ export function BitcoinTokenBalance({
       fiatBalance={fiatBalance}
       availableBalance={availableBalance}
       onPress={onPress}
+      {...rest}
     />
   );
 }
 
 export function BitcoinBalance() {
   const { availableBalance, fiatBalance } = useWalletTotalBitcoinBalance();
-  return <BitcoinTokenBalance availableBalance={availableBalance} fiatBalance={fiatBalance} />;
+  return (
+    <BitcoinTokenBalance
+      availableBalance={availableBalance}
+      fiatBalance={fiatBalance}
+      px="5"
```

Comment:
Maybe we can set `px="5"` + `py="3"` in `TokenBalance` if it's almost always needed? 

Then it can be overwritten when it's not (like in `BitcoinOutcome` )